docs/book/src/plugins/kustomize-v2-alpha.md

```diff
@@ -0,0 +1,111 @@
+# Kustomize v2-alpha Plugin
+
```

Comment:
See here the documentation to better understand it.
Note that after we have this plugin, tools such as SDK will also be able to allow its users try out kustomize v4 and its languages plugins Helm/Ansible using the new mac arch, doing things like:

```
operator-sdk init --plugins=kustomize/v2-alpha,base.operatorframework.helm.sdk/v1
``` 
Then, we can provide all support and advantages to the users without introducing breaking changes.
So that, when the plugins become mature and we no longer need to support k8s versions < 1.22 then we might want to have new languages plugins that adopt them by default. 

c/c @jmrodri